src/reference-manual/expressions.qmd

```diff
@@ -417,6 +418,13 @@ zero-element array.
 array[0] int a;   // a is fully defined as zero element array
 ```
 
+If needed, it is also possible to create an empty array with
+```stan
+rep_array(0, 0)
+```
+where the type of the first argument determines the type of the array
+(e.g., real or int).
```

Comment:
The real and int should be escaped as `real` and `int`. (back ticks)

Reply:
I think it'd be worth pulling out a subsection and highlighting the empty array can be any type, not just a scalar.


### Empty arrays {-}

The empty array expression (`{ }`) is not allowed. See more about restrictions on array expressions in subsection [Restrictions on values](#array-expressions).


If needed, it is also possible to create an empty array with the expression

```stan
rep_array(e, 0)
```
where the first expression `e` determines the type of the array.  For example, `rep_array(0.0, 0)` returns an empty real array of type `real[]`, whereas `rep_array({123}, 0)` returns an empty two dimensional integer array of type `int[ , ]`.  Only the type of the first argument is used, so the integer arrays `{123}` and `{0}` produce equivalent values.
```

Reply:
Github shows the markdown in the comment so, that I can't see how many `#` you used for the subsection "Empty arrays". Also I'm not certain where you would like to have that subsection.